converters/fromZigbee.js

```diff
@@ -5628,22 +5535,6 @@ const converters = {
             }
         },
     },
-    SJCGQ11LM_temperature: {
```

Comment:
SJCGQ11LM is missing from https://github.com/Koenkk/zigbee-herdsman-converters/blob/530b629569428661ccf5cc207a4ac490116648a5/lib/xiaomi.js#L159

Reply:
Same than above, the condition is negative, so it will enter here.

Reply:
whoops, missed this!